src/inclusion_proof.rs

```diff
@@ -27,6 +27,10 @@ pub fn generate_inclusion_proof(
     start_block: u64,
     end_block: u64,
 ) -> Result<Vec<[FixedBytes<32>; 15]>, EraValidateError> {
+    if start_block > end_block {
```

Comment:
On flat head there is also the same check I believe, it is redundant but it is necessary if header_accumulator is to be used with something else